ximea_ros2_cam/include/ximea_ros2_cam/ximea_ros2_cam.hpp

```diff
@@ -158,6 +160,10 @@ class XimeaROSCam : public rclcpp::Node, std::enable_shared_from_this<XimeaROSCa
    bool cam_framerate_control_;   // framerate control - enable or disable
    int cam_framerate_set_;      // framerate control - setting fps
    int cam_img_cap_timeout_;       // max time to wait for img
+   std::string cam_downsampling_;
```

Comment:
document each attribute

Reply:
done

Reply:
Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/pull/1/commits/2bb6e0eae9117872aa357f1acf15b0cc064acba2